frontend/conf/routes

```diff
@@ -86,6 +86,9 @@ POST           /feedback                          controllers.Info.submitFeedbac
 GET            /about                             controllers.Info.about
 GET            /patrons                           controllers.Info.patron
 
+# Offer
+GET            /offer/subscriber                  controllers.Offer.subscriber
```

Comment:
As discussed, I'm not a huge fan of this kind of URL structure but it's consistent with the rest of the app so ¯\(°_o)/¯